src/tests/ftest/cart/iv_server.c

```diff
@@ -863,6 +863,7 @@ fetch_done(crt_iv_namespace_t ivns, uint32_t class_id,
 		if (keys_equal(iv_key, &entry->key) == true) {
 			rc = crt_bulk_create(g_main_ctx, &entry->value,
 					     perms, &bulk_hdl);
+			assert(rc == 0);
```

Comment:
The surrounding code uses D_ASSERT for this test, which places a message into the log file.  Shouldn't you be using this also?

Reply:
@SABollinger the code in this test actually uses assert() I think. wondering if it could be apart of a different cleanup effort to change all of the asserts to D_ASSERT(). This patch was just a quick fix to get the SDL defect resolved for 1.2.